go.mod

```diff
@@ -1 +1,5 @@
 module github.com/buildkite/interpolate
+
+go 1.12
+
+require github.com/drone/envsubst v1.0.2
```

Comment:
Did you mean to require this?

Reply:
Yeah, its used by `substitutions.go` 

The official Go behaves a bit differently, here is the diff:

```
diff --git i/this.go w/this.go
index d39d244..9306b0c 100644
--- i/this.go
+++ w/this.go
@@ -6,14 +6,13 @@ package path
 
 import (
 	"errors"
-	"strings"
 	"unicode/utf8"
 )
 
-// ErrBadPattern indicates a pattern was malformed.
+// ErrBadPattern indicates a globbing pattern was malformed.
 var ErrBadPattern = errors.New("syntax error in pattern")
 
-// Match reports whether name matches the shell pattern.
+// Match reports whether name matches the shell file name pattern.
 // The pattern syntax is:
 //
 //	pattern:
@@ -43,7 +42,10 @@ Pattern:
 		star, chunk, pattern = scanChunk(pattern)
 		if star && chunk == "" {
 			// Trailing * matches rest of string unless it has a /.
-			return !strings.Contains(name, "/"), nil
+			// return !strings.Contains(name, "/"), nil
+
+			// Return rest of string
+			return true, nil
 		}
 		// Look for match at current position.
 		t, ok, err := matchChunk(chunk, name)
@@ -59,8 +61,7 @@ Pattern:
 		}
 		if star {
 			// Look for match skipping i+1 bytes.
-			// Cannot skip /.
-			for i := 0; i < len(name) && name[i] != '/'; i++ {
+			for i := 0; i < len(name); i++ {
 				t, ok, err := matchChunk(chunk, name[i+1:])
 				if ok {
 					// if we're the last chunk, make sure we exhausted the name
@@ -158,9 +159,6 @@ func matchChunk(chunk, s string) (rest string, ok bool, err error) {
 			}
 
 		case '?':
-			if s[0] == '/' {
-				return
-			}
 			_, n := utf8.DecodeRuneInString(s)
 			s = s[n:]
 			chunk = chunk[1:]
```